seed/tests/test_analyses_views.py

```diff
@@ -236,3 +249,262 @@ def test_retrieve_view_with_no_output_file(self):
+        response = self.client.delete(url, content_type='application/json')
+        assert response.status_code == 404
+
+        # root users can create column in root
+        self.login_as_root_member()
+        response = self.client.delete(url, content_type='application/json')
+        assert response.status_code == 200
+
+    def test_analysis_start(self):
+        url = reverse_lazy('api:v3:analyses-start', args=[self.root_analysis.pk]) + "?organization_id=" + str(self.org.id)
+
+        # child user cannot
+        self.login_as_child_member()
+        response = self.client.post(url, content_type='application/json')
+        assert response.status_code == 404
+
+        # root users can create column in root
+        self.login_as_root_member()
+        response = self.client.post(url, content_type='application/json')
+        assert response.status_code == 200
+
+    def test_analysis_stop(self):
+        url = reverse_lazy('api:v3:analyses-stop', args=[self.root_analysis.pk]) + "?organization_id=" + str(self.org.id)
+
+        # child user cannot
+        self.login_as_child_member()
+        response = self.client.post(url, content_type='application/json')
+        assert response.status_code == 404
+
+        # root users can create column in root
+        self.login_as_root_member()
+        response = self.client.post(url, content_type='application/json')
+        assert response.status_code == 200
+
+    def test_analysis_progress_key(self):
+        url = reverse_lazy('api:v3:analyses-progress-key', args=[self.root_analysis.pk]) + "?organization_id=" + str(self.org.id)
+
+        # child user cannot
+        self.login_as_child_member()
+        response = self.client.get(url, content_type='application/json')
+        assert response.status_code == 404
+
+        # root users can create column in root
+        self.login_as_root_member()
+        response = self.client.get(url, content_type='application/json')
+        assert response.status_code == 200
+
+    def test_analysis_stats(self):
+        url = reverse_lazy('api:v3:analyses-stats') + "?organization_id=" + str(self.org.id) + "&cycle_id=" + str(self.cycle.pk)
+
+        # child user cannot
+        self.login_as_child_member()
+        response = self.client.get(url, content_type='application/json')
+        assert response.status_code == 200
+        assert response.json()["total_records"] == 1
+
+        # root users can create column in root
+        self.login_as_root_member()
+        response = self.client.get(url, content_type='application/json')
+        assert response.status_code == 200
+        assert response.json()["total_records"] == 2
+
+    def test_analysis_get_analyses_for_properties(self):
+        url = reverse_lazy('api:v3:analyses-get-analyses-for-properties') + "?organization_id=" + str(self.org.id)
+        params = json.dumps({"property_ids": [self.root_property.pk, self.child_property.pk]})
+
+        # child user cannot
```

Comment:
can

Reply:
oh! this test shouldn't exist- we are deleting that endpoint.